marlowe/pab/Main.hs

```diff
@@ -51,6 +53,7 @@ instance Pretty Marlowe where
     pretty = viaShow
 handleMarloweContract ::
     ( Member (Error PABError) effs
+    , Member (LogMsg (PABMultiAgentMsg (Builtin Marlowe))) effs
```

Comment:
Where is this effect used?

Reply:
It's used by `Builtin.handleBuiltin`, to log the new messages in `lastLogs`. Before this PR the messages weren't logged with the `LogMsg` effect at all, they just existed somewhere in the contract state.